eng/Versions.props

```diff
@@ -64,82 +64,82 @@
-    <MicrosoftExtensionsLoggingConsoleVersion>6.0.0-alpha.1.20560.10</MicrosoftExtensionsLoggingConsoleVersion>
-    <MicrosoftExtensionsLoggingDebugVersion>6.0.0-alpha.1.20560.10</MicrosoftExtensionsLoggingDebugVersion>
-    <MicrosoftExtensionsLoggingEventSourceVersion>6.0.0-alpha.1.20560.10</MicrosoftExtensionsLoggingEventSourceVersion>
-    <MicrosoftExtensionsLoggingEventLogVersion>6.0.0-alpha.1.20560.10</MicrosoftExtensionsLoggingEventLogVersion>
-    <MicrosoftExtensionsLoggingVersion>6.0.0-alpha.1.20560.10</MicrosoftExtensionsLoggingVersion>
-    <MicrosoftExtensionsLoggingTraceSourceVersion>6.0.0-alpha.1.20560.10</MicrosoftExtensionsLoggingTraceSourceVersion>
-    <MicrosoftExtensionsOptionsConfigurationExtensionsVersion>6.0.0-alpha.1.20560.10</MicrosoftExtensionsOptionsConfigurationExtensionsVersion>
-    <MicrosoftExtensionsOptionsDataAnnotationsVersion>6.0.0-alpha.1.20560.10</MicrosoftExtensionsOptionsDataAnnotationsVersion>
-    <MicrosoftExtensionsOptionsVersion>6.0.0-alpha.1.20560.10</MicrosoftExtensionsOptionsVersion>
-    <MicrosoftExtensionsPrimitivesVersion>6.0.0-alpha.1.20560.10</MicrosoftExtensionsPrimitivesVersion>
-    <MicrosoftExtensionsInternalTransportVersion>6.0.0-alpha.1.20560.10</MicrosoftExtensionsInternalTransportVersion>
-    <SystemComponentModelAnnotationsVersion>6.0.0-alpha.1.20560.10</SystemComponentModelAnnotationsVersion>
-    <SystemDiagnosticsDiagnosticSourceVersion>6.0.0-alpha.1.20560.10</SystemDiagnosticsDiagnosticSourceVersion>
-    <SystemDiagnosticsEventLogVersion>6.0.0-alpha.1.20560.10</SystemDiagnosticsEventLogVersion>
-    <SystemDirectoryServicesProtocolsVersion>6.0.0-alpha.1.20560.10</SystemDirectoryServicesProtocolsVersion>
-    <SystemDrawingCommonVersion>6.0.0-alpha.1.20560.10</SystemDrawingCommonVersion>
-    <SystemIOPipelinesVersion>6.0.0-alpha.1.20560.10</SystemIOPipelinesVersion>
-    <SystemNetHttpJsonVersion>6.0.0-alpha.1.20560.10</SystemNetHttpJsonVersion>
-    <SystemNetHttpWinHttpHandlerVersion>6.0.0-alpha.1.20560.10</SystemNetHttpWinHttpHandlerVersion>
-    <SystemNetWebSocketsWebSocketProtocolVersion>6.0.0-alpha.1.20560.10</SystemNetWebSocketsWebSocketProtocolVersion>
-    <SystemReflectionMetadataVersion>6.0.0-alpha.1.20560.10</SystemReflectionMetadataVersion>
-    <SystemResourcesExtensionsVersion>6.0.0-alpha.1.20560.10</SystemResourcesExtensionsVersion>
-    <SystemRuntimeCompilerServicesUnsafeVersion>6.0.0-alpha.1.20560.10</SystemRuntimeCompilerServicesUnsafeVersion>
+    <MicrosoftExtensionsDependencyModelVersion>6.0.0-alpha.1.20567.3</MicrosoftExtensionsDependencyModelVersion>
+    <MicrosoftNETCoreAppRefVersion>6.0.0-alpha.1.20567.3</MicrosoftNETCoreAppRefVersion>
+    <MicrosoftNETCoreAppRuntimewinx64Version>6.0.0-alpha.1.20567.3</MicrosoftNETCoreAppRuntimewinx64Version>
+    <MicrosoftNETCoreBrowserDebugHostTransportVersion>6.0.0-alpha.1.20567.3</MicrosoftNETCoreBrowserDebugHostTransportVersion>
```

Comment:
Getting consistent errors using this version in global.json. Has dotnet/runtime stopped producing archives usable with `dotnet-install.*` scripts @Anipik @ericstj @mmitche❔ e.g. https://dev.azure.com/dnceng/public/_build/results?buildId=889015&view=logs&j=b72e85ab-3386-5aa9-6405-3837662d9688&t=42fe3a49-1549-510e-de90-ca27b36d7602&l=71
``` text
dotnet-install: Downloading link: https://dotnetcli.azureedge.net/dotnet/Runtime/6.0.0-alpha.1.20567.3/dotnet-runtime-6.0.0-alpha.1.20567.3-linux-x64.tar.gz
curl: (22) The requested URL returned error: 404 Not Found
```

Reply:
Looks like the archives are available for newer dotnet/runtime builds. Not sure what the issue was.